requirements.txt

```diff
@@ -0,0 +1,6 @@
+setuptools==40.6.2
+numba==0.45.1
+numpy==1.17.1
```

Comment:
I would like to move toward pipenv. I don't like these pinned dependencies. How did you determine the versions? Are they minimum supported. pipenv would solve all of these.